docs/Getting Started/Alpine Linux/Root on ZFS.rst

```diff
@@ -340,7 +340,7 @@ System Installation
 
    - Encrypted:
 
-     Pick a strong password. Once compromised, changing password will not keep your
+     ZFS native encryption is buggy, see `a ZFS developer's comment on this issue`__ and `this spreadsheet of bugs`__.  In short, if you care about your data, don't use native encryption.  A LUKS-based guide has yet to be written. Once compromised, changing password will not keep your
```

Comment:
While you MAY have problems mainly with zfs send with some arguments, on default case it's pretty well working. I use native encryption on my laptop at least for 2-3 years now. I propose to add more specific warning. What do you think about:

```suggestion
     While ZFS native encryption is a production ready feature, it has some corner cases, see `a ZFS developer's comment on this issue`__ and `this spreadsheet of bugs`__.  Once compromised, changing password will not keep your
```